pkg/gui/controllers/helpers/suggestions_helper.go

```diff
@@ -182,16 +196,16 @@ func (self *SuggestionsHelper) GetAuthorsSuggestionsFunc() func(string) []*types
 
 	slices.Sort(authors)
 
-	return FuzzySearchFunc(authors)
+	return FilterFunc(authors, self.c.UserConfig.Gui.UseFuzzySearch())
 }
 
-func FuzzySearchFunc(options []string) func(string) []*types.Suggestion {
+func FilterFunc(options []string, useFuzzySearch bool) func(string) []*types.Suggestion {
```

Comment:
We could make this a method on a helper so that it can itself get the value of self.c.UserConfig.Gui.UseFuzzySearch() and spare each callsite from having to pass it in, though:
* I'm not sure where it best belongs (perhaps search helper, but it feels more generic than that)
* We may want to apply different kinds of filtering in different places (likely not, though yet to be seen)

All up I'm not fussed